tests/unit/steps/test_analyze.py

```diff
@@ -536,3 +537,221 @@ def test_split_apps(exp_control_plane, exp_data_plane):
     control_plane, data_plane = Analysis._split_apps(all_apps)
     assert exp_control_plane == control_plane
     assert exp_data_plane == data_plane
+
+
+def test_min_os_release_apps_release_channel(model):
```

Comment:
there are a lot of asserts for a single unit test, is it necessary to keep them all together?